csharp/ql/test/query-tests/Stubs/All/Test.cs

```diff
@@ -134,6 +134,34 @@ public class Nested : Class9
 
         public Class9.Nested NestedInstance { get; } = new Class9.Nested(1);
     }
+
+    public enum Enum1
```

Comment:
Add a test case with non-int enum?

Reply:
Oh, definitely yes!
and also a testcase for function pointers!